platforms/software/dependency-management/src/main/java/org/gradle/api/internal/artifacts/configurations/DefaultConfiguration.java

```diff
@@ -708,7 +708,7 @@ public ResolverResultsResolutionResultProvider(boolean strict) {
         }
 
         @Override
-        public ResolverResults getTaskDependencyValue() {
+        public ResolverResults getTaskDependencyValue(boolean lenient) {
```

Comment:
This does not seem right to me. Graph resolution is always lenient. `ResolverResults` is  completely lenient and does not eagerly throw exceptions except in extreme cases of graph failures